doc/releases/reef.rst

```diff
@@ -2,9 +2,742 @@
+* doc/radosgw/s3: correct eTag op match tables (`pr#61309 <https://github.com/ceph/ceph/pull/61309>`_, Anthony D'Atri)
+* doc/radosgw: disambiguate version-added remarks (`pr#57141 <https://github.com/ceph/ceph/pull/57141>`_, Zac Dover)
+* doc/radosgw: Improve archive-sync-module.rst (`pr#60853 <https://github.com/ceph/ceph/pull/60853>`_, Anthony D'Atri)
+* doc/radosgw: Improve archive-sync-module.rst more (`pr#60868 <https://github.com/ceph/ceph/pull/60868>`_, Anthony D'Atri)
+* doc/radosgw: s/zonegroup/pools/ (`pr#61557 <https://github.com/ceph/ceph/pull/61557>`_, Zac Dover)
+* doc/radosgw: update Reef S3 action list (`pr#57365 <https://github.com/ceph/ceph/pull/57365>`_, Zac Dover)
+* doc/radosgw: update rgw_dns_name doc (`pr#60886 <https://github.com/ceph/ceph/pull/60886>`_, Zac Dover)
+* doc/radosgw: use 'confval' directive for reshard config options (`pr#57024 <https://github.com/ceph/ceph/pull/57024>`_, Casey Bodley)
+* doc/rbd/rbd-exclusive-locks: mention incompatibility with advisory locks (`pr#58864 <https://github.com/ceph/ceph/pull/58864>`_, Ilya Dryomov)
+* doc/rbd: add namespace information for mirror commands (`pr#60270 <https://github.com/ceph/ceph/pull/60270>`_, N Balachandran)
+* doc/rbd: fix typos in NVMe-oF docs (`pr#58188 <https://github.com/ceph/ceph/pull/58188>`_, N Balachandran)
+* doc/rbd: use https links in live import examples (`pr#61604 <https://github.com/ceph/ceph/pull/61604>`_, Ilya Dryomov)
+* doc/README.md - add ordered list (`pr#59799 <https://github.com/ceph/ceph/pull/59799>`_, Zac Dover)
+* doc/README.md: create selectable commands (`pr#59835 <https://github.com/ceph/ceph/pull/59835>`_, Zac Dover)
+* doc/README.md: edit "Build Prerequisites" (`pr#59638 <https://github.com/ceph/ceph/pull/59638>`_, Zac Dover)
+* doc/README.md: improve formatting (`pr#59786 <https://github.com/ceph/ceph/pull/59786>`_, Zac Dover)
+* doc/README.md: improve formatting (`pr#59701 <https://github.com/ceph/ceph/pull/59701>`_, Zac Dover)
+* doc/releases: add actual_eol for quincy (`pr#61360 <https://github.com/ceph/ceph/pull/61360>`_, Zac Dover)
+* doc/releases: Add ordering comment to releases.yml (`pr#62193 <https://github.com/ceph/ceph/pull/62193>`_, Anthony D'Atri)
+* doc/rgw/d3n: pass cache dir volume to extra_container_args (`pr#59768 <https://github.com/ceph/ceph/pull/59768>`_, Mark Kogan)
+* doc/rgw/notification: persistent notification queue full behavior (`pr#59234 <https://github.com/ceph/ceph/pull/59234>`_, Yuval Lifshitz)
+* doc/rgw/notifications: specify which event types are enabled by default (`pr#54500 <https://github.com/ceph/ceph/pull/54500>`_, Yuval Lifshitz)
+* doc/security: remove old GPG information (`pr#56914 <https://github.com/ceph/ceph/pull/56914>`_, Zac Dover)
+* doc/security: update CVE list (`pr#57018 <https://github.com/ceph/ceph/pull/57018>`_, Zac Dover)
+* doc/src: add inline literals (``) to variables (`pr#57937 <https://github.com/ceph/ceph/pull/57937>`_, Zac Dover)
```

Comment:
Something's off with the markdown formatting here. @anthonyeleven @zdover23, any suggestions on how to fix?